common/src/java/com/zimbra/common/service/ServiceException.java

```diff
@@ -60,6 +60,10 @@
     public static final String FORBIDDEN = "service.FORBIDDEN";
     // generic "not found" error for objects other than mail items
     public static final String NOT_FOUND = "service.NOT_FOUND";
+    public static final String INTERNAL_ERROR = "service.INTERNAL_ERROR";
```

Comment:
I think we need to add translations and user readable strings for all these errors on client side as well
can you check what happens when client receives one of these errors?

Reply:
Tried reproducing on the UI, will not be possible to reproduce on the UI as the best possible scenario is to pick up the datasource cases, but it was not possible to send e-mails using the externally configured datasource Id. Opened the following bugs related to that:
[ZCS-10331](https://jira.corp.synacor.com/browse/ZCS-10331)
[ZCS-10332](https://jira.corp.synacor.com/browse/ZCS-10332)